src/components/Comments/Comment.js

```diff
@@ -199,7 +273,7 @@ const Comment = ({
           </ButtonHightlight>
         </TouchableOpacity>
 
-        <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
+        {totalVote != 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering of the total vote count based on its value is a good practice to enhance UI clarity.



<!-- This is an auto-generated comment by OSS CodeRabbit -->